tests/Estina/SmartFocus/Tests/Api/Rest/MemberTest.php

```diff
@@ -67,7 +65,7 @@ public function testInsertMemberByEmailAddress()
         $client = $this->getHiddenProperty($service, 'client');
         $client->expects($this->once())
                ->method('get');
-        $response = $service->insertMemberByEmailAddress($this->token, $this->email);
+        $service->insertMemberByEmailAddress($this->token, $this->email);
```

Comment:
why not to use `addToAssertionCount` like we did in NotificationTest.php?

Reply:
@ahmedelbougha  this test already asserts something more specific, that the method above is called once